Dockerfile.proto

```diff
@@ -0,0 +1,66 @@
+
+ARG UBUNTU_VERSION=22.04
+ARG TFSERVING_VERSION=2.9.0
```

Comment:
Move and merge with line 8. Or even move it to line 26.

Reply:
In the regular Dockerfile they try to keep all these versions on top, to make it easier to find them in case they need to be step-up (see grpc version in this PR https://github.com/AxisCommunications/acap-runtime/pull/88/files)

Reply:
Ack